frontend/src/modules/Catalog/index.js

```diff
@@ -7,7 +7,8 @@ export const CatalogsModule = {
   resolve_dependency: () => {
     return (
       getModuleActiveStatus(ModuleNames.S3_DATASETS) ||
-      getModuleActiveStatus(ModuleNames.DASHBOARDS)
+      getModuleActiveStatus(ModuleNames.DASHBOARDS) ||
+      getModuleActiveStatus(ModuleNames.DATASETS_BASE)
```

Comment:
minor: `getModuleActiveStatus(ModuleNames.DATASETS_BASE)` will always return false in `datasets_base` not in `config.json` 

does not affect logic though

Reply:
I need to review that, because maybe not now but I am counting on being able to define all datasets enabled like this.... I'll have a second look because by the way ModuleNames and dependencies are defined it should work